crates/sui-core/src/consensus_handler.rs

```diff
@@ -535,6 +561,27 @@ impl<T, C> ConsensusHandler<T, C> {
         VerifiedExecutableTransaction::new_system(transaction, self.epoch())
     }
 
+    fn randomness_state_update_transaction(
+        &self,
+        round: u64,
+        randomness_round: u64,
+        random_bytes: Vec<u8>,
+    ) -> VerifiedExecutableTransaction {
+        debug!("creating randomness state update transaction");
```

Comment:
maybe move this line down and {:?} log the whole tx?

Reply:
done